config/crd/bases/mutations.gatekeeper.sh_assign.yaml

```
@@ -224,6 +226,46 @@ spec:
           type: object
         status:
           description: AssignStatus defines the observed state of Assign
+          properties:
+            byPod:
+              items:
+                description: MutatorPodStatusStatus defines the observed state of
+                  MutatorPodStatus
+                properties:
+                  enforced:
+                    type: boolean
+                  errors:
+                    items:
+                      description: Error represents a single error caught while adding
+                        a constraint to OPA
```

```suggestion
                        a mutator to a system
```
Did `make manifests` not update/regen this?

---

per mmirecki@'s comment, I think this was a result of not updating which `Error` struct I was using, fixing.